operations/src/main/java/edu/unc/lib/boxc/operations/impl/order/ClearOrderValidator.java

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright 2008 The University of North Carolina at Chapel Hill
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package edu.unc.lib.boxc.operations.impl.order;
+
+import edu.unc.lib.boxc.model.api.ResourceType;
+import edu.unc.lib.boxc.model.api.objects.RepositoryObjectLoader;
+import edu.unc.lib.boxc.operations.api.order.OrderValidator;
+import edu.unc.lib.boxc.operations.jms.order.OrderRequest;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author snluong
+ */
+public class ClearOrderValidator implements OrderValidator {
+    private RepositoryObjectLoader repositoryObjectLoader;
+    private OrderRequest request;
+    private Boolean result;
+    private List<String> errors = new ArrayList<>();
+
+    @Override
+    public boolean isValid() {
+        if (result != null) {
+            return result;
+        }
+        result = validate();
+        return result;
+    }
+
+    private boolean validate() {
+        var parentId = request.getParentPid().getId();
+        var parentObj = repositoryObjectLoader.getRepositoryObject(request.getParentPid());
+        if (!ResourceType.Work.equals(parentObj.getResourceType())) {
+            errors.add("Object " + parentId + " of type " + parentObj.getResourceType().name()
+                    + " does not support setting member order");
```

Comment:
Perhaps we should remove the word "setting" here and in the other validator. In both cases we're really validating whether the object supports ordering at all, rather than whether it supports any particular operations (set, clear, etc)

Reply:
makes sense! i changed them both